src/index.ts

```diff
@@ -760,12 +919,11 @@ export async function load(config: ReaderConfig): Promise<any> {
   });
 
   // Highlighter
-  if ((publication.Metadata.Rendition?.Layout ?? "unknown") !== "fixed") {
-    D2Highlighter = await TextHighlighter.create({
-      delegate: D2Navigator,
-      ...config.highlighter,
-    });
-  }
+  D2Highlighter = await TextHighlighter.create({
```

Comment:
No longer need the conditional here for "fixed" layout? I'm not sure if highlighting a fixed layout will actually work, like one of the graphic novel epubs /006294584X